docs/src/api/class-response.md

```diff
@@ -48,6 +48,16 @@ Contains a boolean stating whether the response was successful (status in the ra
 
 Returns the matching [Request] object.
 
+## async method: Response.securityDetails
+- returns: <[null]|[SecurityDetails]>
```

Comment:
Checkout `method: Request.timing` and expand it here, that way you get a chance to document fields and also you won't need overrides.

Reply:
Thanks! Updated to use that strategy.

However, it looks like inline optionals are not supported so I used an explicit union.

This works fine (notice the explicit `[null]|T`)
```
- returns: <[null]|[Object]>
  - `issuer?` <[string]> Common Name component of the Issuer field
```
but (no union, `?` on `Object` type instead):
```
- returns: <[Object?]>
  - `issuer?` <[string]> Common Name component of the Issuer field
```
errors with:
```
$ node utils/generate_types/
Error: Nested properties given, but there are no objects in type expression: [Object?]
    at Function.parse (/Users/rwoll/code/playwright/utils/doclint/documentation.js:421:13)
    at ApiParser.parseType (/Users/rwoll/code/playwright/utils/doclint/api_parser.js:183:31)
    at ApiParser.parseMember (/Users/rwoll/code/playwright/utils/doclint/api_parser.js:90:27)
    at /Users/rwoll/code/playwright/utils/doclint/api_parser.js:50:14
    at visit (/Users/rwoll/code/playwright/utils/markdown.js:366:3)
    at visit (/Users/rwoll/code/playwright/utils/markdown.js:368:5)
    at Object.visitAll (/Users/rwoll/code/playwright/utils/markdown.js:358:5)
    at new ApiParser (/Users/rwoll/code/playwright/utils/doclint/api_parser.js:48:8)
    at parseApi (/Users/rwoll/code/playwright/utils/doclint/api_parser.js:308:10)
    at /Users/rwoll/code/playwright/utils/generate_types/index.js:40:19
```

I was modeling the `?` syntax off:

```
## async method: Response.json
* langs: csharp
- returns: <[JsonElement?]>
```